src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
 Setup For Java
 ===============
+
+For the entirety of this course, we will be coding in Java. We need to not only have
+Java installed on our machines, but also some accompanying technologies to allow
+us to run and edit our Java code.
+
+Java Development Kit
+--------------------
+
+Installing Java means downloading a package of software called the **Java Development Kit**,
+or **JDK**, for short. The JDK contains software the tools needed to develop and
+run Java code, namely the the Java compiler, **javac**, and the 
+**Java Virtual Machine (JVM)**. 
+
+While the compiler is responsible for processing Java code into machine readable
+code, the JVM allows us to run that code on any computer. These tools 
+together, downloaded as the JDK, give us the means to write, compile, and run Java
+on our machines.
+
+A step-by-step walk-through of the process:
+
+We write code in Java, the code is passed through the compiler program, which 
+translates Java into **bytecode**, a language readable by the JVM. In the JVM,
+bytecode is translated to machine code, which is then executed by your computer.
+
+The JVM gives Java more flexibility than other compiled programming languages because
+it will translate bytecode into the appropriate machine code, depending on the 
+operating environment.
+
+
+Install the JDK
+---------------
+
+In order to learn Java, you need to have the JDK installed on your computer. 
+
+Open a terminal window on your machine and enter the following command:
+
+.. sourcecode:: bash
+
+   java -version
+
+If the response returns a version 13 or higher, you can move on to the section below,
+:ref:`cli-java`.
+
+If you do not have a version of Java at 13 or higher, you can download 
+it `here <https://www.oracle.com/technetwork/java/javase/downloads/jdk13-downloads-5672538.html>`__.
+The relevant install link is on the bottom of the page:
+
+.. figure:: figures/installJava.png
+   :alt: Install Java
+
+   Install Java
+
+To install, you must first select *Accept License Agreement*, then select any of 
+the file type options for your operating system. Once you have completed the 
```

Comment:
If the users need to double-click the file after download, then we should add instructions making that point.  Also, not everyone knows the difference between ``tar``, ``dmg``, ``exe``, etc. I'd suggest adding arrows showing the preferred file (i.e. the easiest) to click for each OS.